transcribe/render.py

```diff
@@ -50,12 +57,19 @@ def transcribe(input_path, scribe_type):
                 except ImportError as e:
                     print(e)
                     continue
-                schema = load_schema_file(key)
-                try:
-                    output_doc = \
-                        validate_document(dict(scribe_module_instace),
-                                          schema)
                 except ValueError as e:
                     print(e)
                     continue
-                yield(json.dumps(output_doc, indent=4))
+                except TypeError as e:
+                    print(e)
+                    continue
+                schema = load_schema_file(key)
+                for document in scribe_module_instace.parse():
+                    try:
+                        output_doc = \
+                            validate_document(document,
+                                              schema)
+                    except ValueError as e:
+                        print(e)
+                        continue
+                    yield(json.dumps(output_doc, indent=4))
```

Comment:
This code looks just like the same stanza above.  Can you consider yielding the raw `scribe_module_instance` (spelling fix) document, wrapped by another generator that handles the `validate_document()` call?